docs/self-hosted-appcircle/configure-server/network-access.md

```diff
@@ -156,6 +156,7 @@ If you don't want to enable these URLs or you aren’t comfortable with this, yo
 
 - macOS VM image and the `run.sh` script:
   - storage.googleapis.com/appcircle-dev-common/self-hosted
```

Comment:
@safderun in script we also have other endpoints.

- storage.googleapis.com/storage/v1/b/appcircle-dev-common

Please organize this part of the document separately for `run.sh` and `install-vm.sh`.

Reply:
Done ✅